From 609b9f16a7443e86e55a3cdc76a484253e50267b Mon Sep 17 00:00:00 2001
From: Amit Shah <amit.shah@redhat.com>
Date: Wed, 16 Jun 2010 05:17:38 -0300
Subject: [PATCH 4/5] monitor: Remove pci_del command for Red Hat Enterprise Linux

RH-Author: Amit Shah <amit.shah@redhat.com>
Message-id: <72d5446b79b3ba801f77dd077083d27c186fbb09.1276665204.git.amit.shah@redhat.com>
Patchwork-id: 9928
O-Subject: [RHEL6 PATCH v2 2/3] monitor: Remove pci_del command for Red Hat
	Enterprise Linux
Bugzilla: 602590
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>

pci_del is the old way of hot-unplugging pci (nic or storage) devices.
The newer device_del is the recommended and suggested command.

Signed-off-by: Amit Shah <amit.shah@redhat.com>
---
 qemu-monitor.hx |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 qemu-monitor.hx |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/qemu-monitor.hx b/qemu-monitor.hx
index 0f477da..a6bb1a0 100644
--- a/qemu-monitor.hx
+++ b/qemu-monitor.hx
@@ -867,7 +867,7 @@ STEXI
 Hot-add PCI device.
 ETEXI
 
-#if defined(TARGET_I386)
+#if defined(TARGET_I386) && 0 /* Disabled for Red Hat Enterprise Linux */
     {
         .name       = "pci_del",
         .args_type  = "pci_addr:s",
-- 
1.7.0.3

